debug.xml

```diff
@@ -485,5 +484,7 @@
     <entry key='fifotrack.port'>5124</entry>
     <entry key='smokey.port'>5125</entry>
     <entry key='extremtrac.port'>5126</entry>
+    <entry key='trakmate.port'>5127</entry>
+    <entry key='trakmate.timezone'>19800</entry>
```

Comment:
Even if the device is in another country with different timezone? Then make it a default value in `Context.getConfig().getInteger(...)`.